### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by medsonmoombe**
## The following were Implemented in this project.
- No linter erros
- Check the video[ here](https://www.loom.com/share/e5b6f7ba5f594b55909903cda07aaf61)
- You should personalize the content of your page.

- Colors,
 and Typography: font face, size and weight.
- Layout: composition and space between elements.
- The pages should look almost identical to the original design. 

-  Build only these 2 pages:
  - The home page.
   - The about page.
- Each of these pages should have versions for 2 different screen sizes:

   - Mobile: up to 768px wide.
    - Desktop: 768px or wider.

- The home page should have a link in the menu to the about page.
- The logo in the header links to the home page.
- Mobile menu
- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.

- The mobile menu should have a close (X) button that closes the menu.
- Dynamic page

- The section "Featured speakers" should be created dynamically in JavaScript.
- Use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section 
-  dynamically.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+       <a class="reserve-btn1" href="#">SEE THE WHOLE PROGRAM</a> 
+        </div>
+    </section>
+<div class="main-speakers">
+            <h3 class="spk-heading">Featured Speakers</h3>
+            <hr class="spk-hr">
+            </div> 
+    <section class="spakers-section" id="speakers">
+        
+    </section>
```
Comment (MahmoudBakr23) [inline index.html:117]: - [ ] In the speakers' section, each image has different width and height by default. So, in order to make all the images the same, please add custom widths and heights to align them.

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @medsonmoombe 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linters are correct
- Great design
- Descriptive README file and PR
- Well done using JavaScript

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️

- [ ] Please make sure the navigation to the about page is working properly! Currently, it ends with a 404 page

_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Whoistolu) [reply index.html:117]: - [ ] [REQUIRED] Great job fixing the other recommended changes, however, this needs to be fixed as recommended by the previous reviewer. The speaker images still do not look uniform. Kindly ensure this s fixed.
### Yout implementation
Screenshot from 2022-04-20 17-36-10

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @medsonmoombe 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

- [ ] [REQUIRED] Kindly ensure that your README contains an "Acknowledgements" section that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by medsonmoombe** [comment]
## The following were implemented as the reviewer requested changes
- Update the Readme file to acknowledge the author's design
- Adjust images width and height to match

image

image

**Review by phelian23 — APPROVED** [review]
Hi @medsonmoombe, 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**